spec/controllers/sub_sections_controller_spec.rb

```diff
@@ -10,6 +10,7 @@
   let!(:live_stream) { create :live_stream, :without_validations }
   let(:title) { Faker::Lorem.sentence }
   let(:content) { "###{Faker::Lorem.sentence}" }
+  let(:featured_link) { "/#{SecureRandom.urlsafe_base64}" }
```

Comment:
Is this still needed, as there is a duplicate line at line 83?

Reply:
Good spot! Will remove a fix up.